tomcat/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -x
+while IFS=: read -d: -r path; do
```

Comment:
I’d add two pieces of functionality to this script.

* Use [Go templating](https://github.com/paketo-buildpacks/apache-tomcat/blob/9562b7e4b62af5d9a044d17ab195096352f97842/tomcat/setenv.sh#L2) and the [`sherpa.TemplateFile()`](https://github.com/paketo-buildpacks/apache-tomcat/blob/9562b7e4b62af5d9a044d17ab195096352f97842/tomcat/base.go#L284) to explicitly define the path to the `lib` directory
* Do a `mkdir -p` of the directory you’re symlinking into